packages/vite/src/node/plugins/html.ts

```diff
@@ -29,12 +29,14 @@ import { modulePreloadPolyfillId } from './modulePreloadPolyfill'
 import type {
   AttributeNode,
   NodeTransform,
+  TextNode,
   ElementNode,
   CompilerError
 } from '@vue/compiler-dom'
 import { NodeTypes } from '@vue/compiler-dom'
 
-const htmlProxyRE = /\?html-proxy&index=(\d+)\.js$/
+const htmlProxyRE = /\?html-proxy[&inline]*&index=(\d+)\.(js|css)$/
+const inlineCSSRE = /__VITE_INLINECSS__([^_]+_\d+)__/g
```

Comment:
I found it not obvious at a first glance and it can be misread like `INLINESS`
```suggestion
const inlineCSSRE = /__VITE_INLINE_CSS__([^_]+_\d+)__/g
```